tools/serverpod_cli/pubspec.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
   recase: ^4.0.0
   super_string: ^1.0.3
   watcher: ^1.0.0
-  analyzer: ^3.3.1
+  analyzer: ^4.0.0
```

Comment:
The analyzer package shouldn't be updated in this PR.

Reply:
Done